### DISCUSSION TITLE
rustic 0.5.4

### DISCUSSION THREAD

**Discussion by aawsome** [rustic-rs/rustic#658] [Announcements]
Happy to release rustic 0.5.4.

This is again a small bug-fix release.

Changes in version 0.5.4:

Bugs fixed:
- backup crashed when there was a non-unicode link target. The crash has been fixed. However, non-unicode link targets are still unsupported.
- Extended attributes which were saved with value null couldn't be handled. This has been fixed.
- prune: --max-repack didn't work with a given percentage of repo size. This has been fixed.

New features:
- copy: Added --init option to initialize uninitialized target repos
- dependencies have been updated

<hr /><em>This discussion was created from the release <a href='https://github.com/rustic-rs/rustic/releases/tag/v0.5.4'>rustic 0.5.4</a>.</em>

**Comment by Aderemi-Adesada** [comment]
@aawsome this improvements are awesome. I recently just came across rustic and I would love to contribute to it. I am new to rust and restic, I would love to know the best way I can get started with contributing to rustic. How can I message you privately.

**Comment by aawsome** [comment]
Hi  @Aderemi-Adesada!
Thanks a lot for your interest in rustic and great that you like to contribute!
There are some open issues tagged with "good first issue" - these are perfect points to start. Best you pick one, write a comment in the issue that you would like to start implementing, and then start doing it. Also feel free to ask questions about the implementation and don't hesitate to early open a PR from your branch.

About private messages, there is a discord channel about rustic: https://discord.gg/ckJVq6P4
But note that I'm not often available.

Have fun coding!

**Comment by Aderemi-Adesada** [comment]
thanks

**Comment by aawsome** [comment]
Just forgot one thing: The best starting point, of course, is if there is some functionality you are missing for your personal use cases or if you with some improvements, etc. In that case, please always open an issue - where you can state you already work on it..

**Reply by Aderemi-Adesada** [reply]
thanks, noted
